tests/system/model_monitoring/test_app.py

```diff
@@ -241,14 +270,38 @@ def _test_api_get_values(
             method=mlrun.common.types.HTTPMethod.GET,
             path=f"projects/{cls.project_name}/model-endpoints/{ep_id}/metrics-values{names_query}",
         )
-        for result_values in json.loads(response.content.decode()):
+        results = json.loads(response.content.decode())
+
+        for result_values in results:
             assert result_values[
                 "data"
             ], f"No data for result {result_values['full_name']}"
             assert result_values[
                 "values"
             ], f"The values list is empty for result {result_values['full_name']}"
 
+        result = results[0]
+        assert (
+            result["full_name"]
+            == "test-app-flow-v2.histogram-data-drift.result.general_drift"
```

Comment:
This for sure need not be here. You hard code the project name, but this test is shared between different system tests, and one may choose a different project name.

Reply:
I changed to `cls.project_name`, but it sounds like you would like this assert removed entirely?